Samantha-samanthalpark/Homework-6/500px/app.js

```diff
@@ -0,0 +1,54 @@
+$(function() {
+  // wait to run this code until DOM is now ready
+  _500px.init({
+    sdk_key: 'b60b6dee646e57f949af17874935b35494c12772'
+  });
+
+  $('#login').click(function() {
+    _500px.login();
+  });
+
+  _500px.on('authorization_obtained', function () {
+    //alert('You have logged in');
+    $('.sign-in-view').hide();
+    $('.image-results-view').show();
+
+    if (navigator.geolocation) {
+      navigator.geolocation.getCurrentPosition(function(position) {
+        const lat = position.coords.latitude;
+        const long = position.coords.longitude;
+        console.log(lat + ', ' + long);
+
+        const radius = '10mi';
+
+        const searchOptions = {
+          geo: lat + ',' + long + ',' + radius,
+          only: 'Landscapes',
+          sort: 'highest_rating',
+          rpp: 28,
+        }
+
+        _500px.api('/photos/search', searchOptions, function(response) {
+          if(response.data.photos.length === 0){
+            $('.images').append('No photos found');
+          } else {
+            console.log('Request suceeded!');
+            console.log(response.data.photos);
+            handleSucessResponse(response);
+          }
+        })
+      });
+    } else {
+      $('.images').append('Sorry, your browser does not support geolocation.');
+    }
+  });
+
+  function handleSucessResponse(response){
+    const $photos = response.data.photos;
+
+    $.each($photos, function(){
+      const $newImage = $('<img>').attr('src', $photos.image_url).addClass('image');
```

Comment:
$photos is your data structure, and image_url is the property you want to get from each image. But you'll need to drill down a little deeper into $photos in order to access this property for the current image each time through this loop. I suspect this is why your images weren't showing up in the browser. I suggest using console.log to log the values you're trying to get and then use that to check your work and see if you're getting the results you expect before using that same reference in DOM manipulation. Almost there, though!